### ISSUE TITLE
Bio.Seq import error

### ISSUE THREAD

**Issue by rjavier17** [DavidBSauer/OGT_prediction#9]
Hi. I am trying to run the program but I am facing this error. My system is Ubuntu 22.04. I have tried to run the demonstration files and I get the same error: 

Traceback (most recent call last):
  File "prediction_pipeline.py", line 12, in <module>
    import genome_analysis
  File "/home/tramuso/OGT_prediction/prediction/genome_analysis.py", line 10, in <module>
    import external_tools
  File "/home/tramuso/OGT_prediction/prediction/external_tools.py", line 10, in <module>
    from BCBio import GFF
  File "/usr/local/lib/python3.8/dist-packages/BCBio/GFF/__init__.py", line 3, in <module>
    from BCBio.GFF.GFFParser import GFFParser, DiscoGFFParser, GFFExaminer, parse, parse_simple
  File "/usr/local/lib/python3.8/dist-packages/BCBio/GFF/GFFParser.py", line 34, in <module>
    from Bio.Seq import UnknownSeq
ImportError: cannot import name 'UnknownSeq' from 'Bio.Seq' (/usr/local/lib/python3.8/dist-packages/Bio/Seq.py)

I have installed all the dependencies using the script provided with the software (Ubuntu_setup.bash)

Any hint? What am I missing? 

Thank you very much.

**Comment by DavidBSauer** [comment]
Hello!
Hmmm... This looks like a problem with the BCBio or Biopython packages. Did you get an error when installing these via the Ubuntu_setup.bash script?

I just re-ran the script on a fresh install of Ubuntu 18.04 and everything worked fine. I suggest you double-check the dependencies are correctly installed.

**Comment by rjavier17** [comment]
Thank you very much for your reply!

I started over. I have installed Ubuntu 22.04 under UTM in a Macbook M1, arm64 version (for some reason, 18.04 arm64 won't boot, but as you said, this also should run in 22.04). I don't know if running arm64 version through a VM makes any difference. 

I ran your script and it got stuck installing sklearn, so I manually ran the command to install it along with the other python dependencies (sudo pip3 install numpy scipy matplotlib biopython bcbio-gff tqdm sklearn matplotlib matplotlib-venn). Then, I finally got the 'finished' output. However, I get the same output than before: ImportError: cannot import name 'UnknownSeq' from 'Bio.Seq' (/usr/local/lib/python3.8/dist-packages/Bio/Seq.py).

I am currently trying on a different computer, installing Ubuntu 18.04, AMD64 version. I will try to manually install all the dependencies on Mac. I will keep you posted 😅. 

Thank you again for your help!
